docs/architecture/seo.md

```diff
@@ -49,50 +50,59 @@ export interface PageMeta {
 }
 ```
 
-HTML5 supports a number of specific metatag properties, such as `description`, `keyworks`, etc. These metatags are used by search engines, social platforms and bots. In order to be more specific on one or the other platform, some social platform introduce a specific set of properties. Facebook for example uses an Open Graph protocol that enables any web page to become a rich object in a social graph. Specific metatags can be used to articulate the experience on the social platform. The code snippet below shows a custom page description for facebook.
+HTML5 supports a variety of meta tag properties, such as `description`, `keywords`, and so on. These meta tags are used by search engines, social platforms and bots. Some social platforms have introduced their own sets of properties that are specific to their platforms. For example, Facebook uses an Open Graph protocol that enables any web page to become a rich object in a social graph. Specific meta tags can be used to articulate the experience on the social platform. The code snippet below shows a custom page description for Facebook:
```

Comment:
"HTML5 supports a variety of meta tag properties, such as `description`, `keywords`, and so on."

should be:

"HTML5 supports meta tag properties, such as `description`." (we should not use keywords)

Reply:
Done.